hardware_interface/include/hardware_interface/hardware_component_info.hpp

```diff
@@ -47,6 +48,12 @@ struct HardwareComponentInfo
   /// Component is async
   bool is_async;
 
+  //// read rate
+  unsigned int read_rate;
+
+  //// write rate
+  unsigned int write_rate;
+
```

Comment:
why 2 separate?

Reply:
Good question. In future, if a hardware is slower to write but faster to read. We could have that decoupling already done and it would be easier to extend.

Reply:
Sounds sensible but could we please only do that when we have the need. If there is no immediate need or a date you could put on *when* we need this functionality, I'd leave it out for now.

Reply:
Ok. I can move to a single variable. Not a problem at all.

Reply:
Done

Reply:
Done